test/FunctionalTests/Client/UnaryTests.cs

```diff
@@ -36,7 +36,8 @@ public async Task ThrowErrorWithOK_ClientThrowsFailedToDeserializeError()
     {
         SetExpectedErrorsFilter(writeContext =>
         {
-            if (writeContext.State.ToString() == "Message not returned from unary or client streaming call.")
+            if (writeContext.State.ToString() == "Header contains an OK gRPC status. This is invalid for unary or client streaming calls because a status in the header indicates there is no response body. " +
```

Comment:
Worthwhile to store this message in a constant for use in the tests and implementation?

Reply:
This is the only place it is used in tests. We're not doing it for anything else so I'd like to leave it as is.